lib/media/track/videotrack.js

```diff
@@ -84,18 +109,15 @@ class VideoTrack extends MediaTrack {
 
     this._dummyEl.play().then(() => {
       const captureFrame = cb => {
-        const { enabled, muted, readyState } = this.mediaStreamTrack;
-        if (!enabled || muted || readyState === 'ended') {
-          return;
-        }
-        if (this._dummyEl.requestVideoFrameCallback) {
-          this._dummyEl.requestVideoFrameCallback(cb);
```

Comment:
Are you sure `requestVideoFrameCallback` does not execute when the tab is not in view? The spec did not mention anything about this.

Reply:
Subtrack the processFrame delay for each call.